editor/src/components/navigator/navigator-conditionals.spec.browser2.tsx

```diff
@@ -0,0 +1,303 @@
+import { BakedInStoryboardUID, BakedInStoryboardVariableName } from '../../core/model/scene-utils'
+import {
+  TestSceneUID,
+  renderTestEditorWithCode,
+  EditorRenderResult,
+  getPrintedUiJsCode,
+} from '../../components/canvas/ui-jsx.test-utils'
+import { offsetPoint, windowPoint, WindowPoint } from '../../core/shared/math-utils'
+import { fireEvent } from '@testing-library/react'
+import { selectComponents } from '../editor/actions/meta-actions'
+import * as EP from '../../core/shared/element-path'
+import { act } from 'react-dom/test-utils'
+import {
+  conditionalClauseNavigatorEntry,
+  DerivedState,
+  EditorState,
+  navigatorEntryToKey,
+  regularNavigatorEntry,
+  varSafeNavigatorEntryToKey,
+} from '../editor/store/editor-state'
+import { getDomRectCenter } from '../../core/shared/dom-utils'
+import { FOR_TESTS_setNextGeneratedUids } from '../../core/model/element-template-utils.test-utils'
+import { setFeatureForBrowserTests } from '../../utils/utils.test-utils'
+import { navigatorDepth } from './navigator-utils'
+
+function dragElement(
```

Comment:
Should this function be moved into `event-helpers.test-utils.tsx`?

Reply:
Since this is prep work for tests from a follow up PR, feel free to disregard this question until then